Cargo.toml

```diff
@@ -54,9 +54,54 @@ async-tokio = ["tokio"]
 ## UTF-16 will not work (therefore, `quick-xml` is not [standard compliant]).
 ##
 ## List of supported encodings includes all encodings supported by [`encoding_rs`]
-## crate, that satisfied the restriction above.
+## crate, that satisfied the restriction above. So, the following encodings are
+## **not supported**:
+## - [UTF-16BE]
+## - [UTF-16LE]
+## - [ISO-2022-JP]
+##
+## You should stop to process document when one of that encoding will be detected,
```

Comment:
Yes, I just want to explicitly warn users about that UTF-16 and ISO-2022-JP are not supported for now. For example, parsing documents with some Chinese characters that represented as `[ASCII byte, some byte]` in UTF-16BE or `[some byte, ASCII byte]` in UTF16LE can confuse the parser. That can be avoided if you stop to processing such documents in the very beginning.

I want make this change because I want to cut release this weekend, and it is still unclear when the correct solution will be ready, so it is best to avoid using problematic encodings for now.

I hope that in the next release this note will be removed. It should not break anything -- once correct support will be implemented, users will be updated and remove their guard code.

Reply:
I added a remark, that restriction is temporary and will be eliminated once #158 is fixed.

Reply:
Thanks.  I do think finishing #158 this weekend is plausible but I won't promise it.

Reply:
Well, an intermediate stage of it, in any case.

Reply:
Or it would be, except for async.  Goddammit.  Keep forgetting about that.